### DISCUSSION TITLE
Is there a way to log php errors in ray ?

### DISCUSSION THREAD

**Discussion by valentin-pellegrin** [spatie/ray#726] [Q&A] [answered]
When working with logs, native PHP errors are essential and often logged in a specific file. 
If we can't have php logs in ray we need 2 different tools for log management.

**Comment by ndberg** [comment]
At least in the Laravel implementation, you can enable Log | Error output in the ray config ray.php:
```
'send_log_calls_to_ray' => env('SEND_LOG_CALLS_TO_RAY', true),
'send_exceptions_to_ray' => env('SEND_EXCEPTIONS_TO_RAY', true),
```

**Comment by jmslbam** [comment, marked as answer]
@valentin-pellegrin could this help you? 

Cuz I wrote this, this week https://github.com/jmslbam/ray-php-errors which logs the PHP errors

**Reply by valentin-pellegrin** [reply]
@jmslbam Thanks a lot for your work! It's working for me!

**Reply by jmslbam** [reply]
Glad it works and even after almost 3 years still usefull 😄

**Reply by freekmurze** [reply]
This looks very. Good work! I'd accept a PR that adds this code to Ray itself 👍

**Reply by jmslbam** [reply]
Hi @freekmurze, thanks! Upcomming month I'm totally swamped but will research and schedule it in May 👍
